recipes/dioptas/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "Dioptas" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "5911464f2141f19720d8b987796d86344602737c9e4c34309d0b0dfd2b32d90b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Dioptas/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+requirements:
+  build:
+    -  {{ compiler('fortran') }}
```

Comment:
it seems that you also need 
```
- {{ compiler('c') }}
```

Reply:
hm it seems to work on mac and windows like this. however it has a problem with pyfai.

is this maybe because pyfai is using toolchain? Does toolchain also include a fortran compiler?

Reply:
So the major problem is that libflang (the fortran compiler) only works with vc 14 on windows. Thus it cannot be used with python 2.7 or pyfai which are both compiled with vc 9. Is this solvable somehow?